test/e2e/helpers.go

```diff
@@ -482,10 +482,22 @@ func getProxiedSSHClient(controlPlaneEndpoint, hostname, port string) (*ssh.Clie
 		return nil, errors.Wrapf(err, "dialing from control plane to target node at %s", hostname)
 	}
 
-	// Establish an authenticated SSH conn over the client -> control plane -> target transport
```

Comment:
I think what we're trying to do here is the same as adding the following:
```go
	err = c.SetDeadline(time.Now().Add(1 * time.Minute))
	if err != nil {
		return nil, err
	}
```

That timeout would be for the whole SSH command, so not sure what the best value for that would be. One random successful e2e run I looked at shows all of the Windows log collection commands together taking about 2.5m.